task1.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Aug 27 16:34:28 2018
+
+@author: Tommy
+"""
+import math
+import numpy
+import matplotlib
+
+x = float(input('Please input the width of the gap: '))
+y = float(input('please input the distance you wanna go: '))
+
+Min_angle_alpha = math.atan(x/(y))
+Max_angle_alpha = math.pi/2
+
+alpha = numpy.linspace(Min_angle_alpha, Max_angle_alpha, 100)
```

Comment:
limit the human calculation out of the the program, use pure computer calculation